shared/src/main/scala/squants/mass/Mass.scala

```diff
@@ -171,6 +182,45 @@ object SolarMasses extends MassUnit {
   val symbol = "M☉"
 }
 
+object ElectronVolt extends MassUnit {
```

Comment:
Won't this cause name collisions in the REPL since we import all constants and implicits by default?